integration_test/tests/common/mod.rs

```diff
@@ -425,3 +425,36 @@ pub fn get_column_attributes(stmt: Handle, expected_col_count: SmallInt) {
         }
     }
 }
+
+#[allow(dead_code)]
+/// Helper function to bind columns.
+pub unsafe fn bind_cols(stmt_handle: HStmt, target_types: Vec<(CDataType, Pointer, Len)>) {
+    for (i, (target_type, binding_buffer, buffer_length)) in target_types.iter().enumerate() {
+        assert_eq!(
+            SqlReturn::SUCCESS,
+            SQLBindCol(
+                stmt_handle,
+                (i + 1) as USmallInt,
+                *target_type as SmallInt,
+                *binding_buffer,
+                *buffer_length,
+                null_mut(),
```

Comment:
The failures are due to a segmentation fault, and here we can see that the `length_or_indicator` is null, so that's probably the cause of the problem.

Reply:
Updated!